features/bootstrap/FeatureContext.php

```diff
@@ -332,11 +332,45 @@ public function getHookDefinitionResources() {
 	 * Replace {VARIABLE_NAME}. Note that variable names can only contain uppercase letters and underscores (no numbers).
```

Comment:
The comment about variable name requirements is not technically correct anymore.